### PR TITLE
Figure texts and small plot bugs

### PR DESCRIPTION
**Pull request by ThereseBjorkman**
Figure texts placeholders added for all the graphs as well as fixing a bug connected to the haplotype graph that made it possible to insert invalid individual id and breakiing the system.

### CODE REVIEW

**Comment by hannamlmv** [comment]
This should probably be merged before we do the UI testing

File: src/tseda/vpages/ignn.py [hunk 1]
```diff
@@ -41,7 +41,6 @@ class GNNHaplotype(View):
 
     individual_id = param.Integer(
         default=None,
-        bounds=(0, None),
```
Comment (percyfal) [inline src/tseda/vpages/ignn.py:44]: Why has the lower bound been removed? We know that indices are >= 0. Also, I wonder if we should set the upper bound to the (known) number of individuals.

File: src/tseda/vpages/ignn.py [hunk 2]
```diff
@@ -122,25 +127,52 @@ def plot(self, haplotype=0):
...
+                self.plot(1),
+            )
+        else:
+            return pn.Column(pn.pane.Markdown(""))
 
     def sidebar(self):
         return pn.Card(
             self.param.individual_id,
             self.param.window_size,
+            self.position_index_warning,
```
Comment (percyfal) [inline src/tseda/vpages/ignn.py:175]: I think renaming `position_index` to `individual_id` would make sense in this module

**Review by percyfal — CHANGES_REQUESTED** [review]

File: src/tseda/vpages/ignn.py (hunk 1 above)
Comment (ThereseBjorkman) [reply src/tseda/vpages/ignn.py:44]: Because the lower bound made it so that if a negative number is entered panel automatically showed the tree for index zero. I thought it would be better to get a warning message when negative numbers are entered to avoid confusion.

File: src/tseda/vpages/ignn.py (hunk 2 above)
Comment (ThereseBjorkman) [reply src/tseda/vpages/ignn.py:175]: Fixing!

File: requirements-dev.lock [hunk 3]
```diff
@@ -17,6 +17,8 @@ anyio==4.4.0
 appdirs==1.4.4
     # via stdpopsim
     # via tseda
+appnope==0.1.4
+    # via ipykernel
```
Comment (hannamlmv) [inline requirements-dev.lock:21]: why was this added? maybe remove it if unnecessary?

File: src/tseda/vpages/ignn.py [hunk 4]
```diff
@@ -122,25 +127,52 @@ def plot(self, haplotype=0):
...
-        return pn.Column(
-            pn.pane.Markdown(f"## Individual id {self.individual_id}"),
-            self.warning_pane,
-            pn.pane.Markdown(f"### Haplotype 0 (sample id {nodes[0]})"),
-            self.plot(0),
-            pn.pane.Markdown(f"### Haplotype 1 (sample id {nodes[1]})"),
-            self.plot(1),
-        )
+            return pn.pane.Markdown(
+                "**Enter a valid sample id to see the GNN haplotype plot.**"
```
Comment (hannamlmv) [inline src/tseda/vpages/ignn.py:152]: Maybe we should still display the heads "GNN Haplotype plot" so that the user understands what plot this message is connected to, i.e. not the GNN VBar plot

**Review by hannamlmv — CHANGES_REQUESTED** [review]

File: src/tseda/vpages/ignn.py (hunk 4 above)
Comment (ThereseBjorkman) [reply src/tseda/vpages/ignn.py]: Done!

File: src/tseda/vpages/ignn.py [hunk 5]
```diff
@@ -56,9 +55,15 @@ class GNNHaplotype(View):
         visible=False,
     )
 
+    position_index_warning = pn.pane.Alert(
```
[the remark this answers is not in the document]
Comment (ThereseBjorkman) [reply src/tseda/vpages/ignn.py]: Fixed!
